kuttl-test.yaml

```diff
@@ -23,6 +23,6 @@ crdDir: config/crd/bases
 manifestDirs:
   - tests/resources/crds
 commands:
-  - command: build/cartographer
+  - command: build/cartographer --log-level debug
```

Comment:
👍 

Separate future chore: I want to figure out how to pass a flag to the make test commands to quiet the output of kuttl tests. Indeed, that might point to a reason to include a `--log-level fatal` so that even the error logs are suppressed.

Reply:
actually we probably dont want to leave this in. It's simple enough to change when you need it. To your question, i think the only ways are to pass a different kuttl-test.yaml OR use env vars. If we have a `CARTO_LOG_LEVEL` set in our shell (or by make) it _should_ pass all the way into the binary.

Reply:
👍 

https://github.com/peterbourgon/ff

makes that pretty easy for plain flags